evaluate_rag.py

```diff
@@ -11,6 +13,8 @@
 from langchain_core.embeddings import Embeddings
 from langchain_core.language_models import BaseChatModel
 from langchain_core.vectorstores import VectorStore
+from langchain.retrievers import BM25Retriever
```

Comment:
this will be deprecated soon, let's change to `from langchain_community.retrievers import BM25Retriever`

```
LangChainDeprecationWarning: Importing this retriever from langchain is deprecated. Importing it from langchain will no longer be supported as of langchain==0.2.0. Please import from langchain-community instead:

`from langchain_community.retrievers import BM25Retriever`.
```